src/github.com/matrix-org/dendrite/clientapi/writers/createroom.go

```diff
@@ -176,7 +177,19 @@ func createRoom(req *http.Request, cfg config.ClientAPI, roomID string) util.JSO
 		// Add the event to the list of auth events
 		builtEventMap[common.StateKeyTuple{e.Type, e.StateKey}] = ev
 		builtEvents = append(builtEvents, ev)
+	}
 
+	// send events to the room server
+	var m sarama.ProducerMessage
+	value, err := json.Marshal(builtEvents)
```

Comment:
I *think* my plan was that the "m.room.create" event should be KindOutlier because the server isn't in the room until it gets a m.room.member join event. The m.room.member join event should be KindJoin with the m.room.create as the state. All the other events should be KindNew.